release_package.yml

```diff
@@ -10,8 +10,30 @@
   any_errors_fatal: false # don't bomb out the entire playbook if one host (i.e. package) fails
   gather_facts: no
   tasks:
-    - include: tasks/setup_sources.yml
+    - name: 'Get latest package info from copr'
+      command: "copr-cli get-package @theforeman/tfm-ror51 --name {{ inventory_hostname }} --with-latest-build"
+      register: copr_package_info
 
-    - include: tasks/tito_release.yml
+    - name: 'Get latest package info from git'
+      command: "rpmspec --query --define='dist .el7.centos' --queryformat=%{version}-%{release} --srpm packages/{{ inventory_hostname }}/{{ inventory_hostname }}.spec"
+      register: git_package_info
 
-    - include: tasks/build_wait_and_download.yml
+    - set_fact:
+        git_package_version: "{{ git_package_info.stdout }}"
+
+    - set_fact:
+        copr_package_info: "{{ copr_package_info.stdout | from_json }}"
+
+    - set_fact:
+        copr_package_version: "{{ copr_package_info['latest_build']['pkg_version'] }}"
+
+    - debug:
+        msg: "Git version:  {{ git_package_version }} \nCopr version: {{ copr_package_version }}"
```

Comment:
there is a double space before `{{ git_package_version }}`

Reply:
That is actually intentional, mostly just to align for better visual inspection.